pubtools/_pulp/tasks/push/command.py

```diff
@@ -144,16 +148,19 @@ def add_phase(klass, **kwargs):
             add_phase(Update)
 
             # Ensure all items are associated into the desired target repos.
-            add_phase(Associate)
+            associate_phase = add_phase(Associate)
 
             if "publish" in self.args.skip:
-                # Caller doesn't want to publish, then we just wait for prior phases
-                # to complete.
+                # Caller doesn't want to publish, then we just do post push actions
+                # for push items and wait for prior phases to complete.
+                add_phase(PostPushActions)
```

Comment:
Similarly to the pre-push case, are you sure you want to invoke the hook if publish didn't happen?  Or do you want to add an argument onto the hook which says whether or not publish happened, so the other side can tell?